day5app/src/App.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import * as firebase from "firebase";
+import Table from './Table.js'
+
+var config = {
+    apiKey: "AIzaSyC4aPMurZ3Boaun5byKQX1XJ-xoMDO2V_0",
```

Comment:
You should **never** commit keys, passwords, or tokens. Check out how to use config.js files with node [environment variables](https://nodejs.org/api/process.html#process_process_env).